apps/poe/poe.yml

```diff
@@ -0,0 +1,25 @@
+name: Poe
+description: 'Distraction-free writing app, built by writers for writers'
+website: https://getpoe.com
+category: Productivity
+keywords:
+  - poe
+  - poe-writer
+  - focus
+  - writing
+  - book
+  - author
+  - write
+  - editor
+  - novel
+  - 'distraction free writing'
+  - markdown
+  - win
+  - mac
```

Comment:
I didn't find the download address for mac on the official website

Reply:
mac version failed to build on last release, but working now